src/backend/electron/main.ts

```diff
@@ -113,7 +113,17 @@ if (errorForRemoveBeforeUserDataDir != undefined) {
 let win: BrowserWindow;
 
 process.on("uncaughtException", (error) => {
-  log.error(error);
+  const stack = error.stack ? error.stack : `${error.name}: ${error.message}`;
+  const message = "Uncaught Exception:\n" + stack;
+  log.error(message);
+  if (isDevelopment) {
+    app.exit(1);
+  } else {
+    dialog.showErrorBox(
+      "メインプロセスで原因不明のエラーが発生しました",
+      message,
+    );
+  }
```

Comment:
こんな感じとかどうでしょう？
（良さそうとか適当にコメントいただければ適用してマージします！！）

```suggestion
  log.error(error);

  if (isDevelopment) {
    app.exit(1);
  } else {
    const { message, name } = error;
    let detailedMessage = "";
    detailedMessage += `メインプロセスで原因不明のエラーが発生しました。\n`;
    detailedMessage += `エラー名: ${name}\n`;
    detailedMessage += `メッセージ: ${message}\n`;
    if (error.stack) {
      detailedMessage += `スタックトレース: \n${error.stack}`;
    }

    dialog.showErrorBox("エラー", detailedMessage);
  }
```

本当は報告があるように問い合わせ先みたいなのも書こうと思ったんですが･･･
ちょっと問い合わせ経路が微妙（「お問い合わせ」「エラー報告」に案内するか、直接「ツイッターハッシュタグか`@voicevox_pj`に連絡して」と伝える形）なので一旦いいかなぁ。。

Reply:
多分大丈夫だと思います。